conf/routes

```diff
@@ -24,6 +24,7 @@ POST        /consignment/:consignmentId/transfer-agreement-continued
 GET         /consignment/:consignmentId/upload                                                    controllers.UploadController.uploadPage(consignmentId: java.util.UUID)
 POST        /start-upload                                                                         controllers.UploadController.startUpload()
 POST        /save-metadata                                                                        controllers.UploadController.saveClientMetadata()
+POST        /consignment/:consignmentId/draft-metadata                                            controllers.DraftMetadataUploadController.saveDraftMetadata(consignmentId: java.util.UUID)
```

Comment:
Is it possible to change this route to `/consignment/:consignmentId/draft-metadata/upload`?

Reply:
@ian-hoyle Is this possible?